src/main/scala/cromwell/engine/callexecution/CallExecutionActor.scala

```diff
@@ -88,22 +75,35 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
     }
   }
 
+  /**
+    * Update the ExecutionHandle
+    */
+  def pollerFunction(handle: ExecutionHandle): Future[ExecutionHandle]
```

Comment:
Can this and the below be `poll` and `execute` respectively?  The "function" part seems self-explanatory.  :smile: